src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -92,24 +92,40 @@ export class Accept extends Component<AcceptProps, AcceptState> {
             const {
               ecommerceBuyerAcceptOffer: { orderOrError },
             } = data
-
-            if (orderOrError.error) {
-              this.onMutationError(
-                new ErrorWithMetadata(
-                  orderOrError.error.code,
-                  orderOrError.error
-                )
-              )
-            } else {
-              this.props.router.push(`/orders/${this.props.order.id}/status`)
-            }
+            this.onSubmitCompleted(orderOrError)
           },
           onError: this.onMutationError.bind(this),
         })
       }
     })
   }
 
+  onSubmitCompleted(orderOrError) {
```

Comment:
Huh, confusing to see these changes in here since they're from https://github.com/artsy/reaction/pull/1699.